FtpClientApp/FtpClientApp/Program.cs

```diff
@@ -184,23 +184,26 @@ public static bool GetResponce(String username, String password, String server)
                     MyAnswer = false;
                     break;
                 case "3":
-                    Console.WriteLine(" Not Implemented Yet \n");
+                    Console.WriteLine(" You choose 3, List Directories, in devl \n");
                     //list remote directory
+                    ListFiles ListRemoteFiles = new ListFiles(conn);
                     MyAnswer = false;
                     break;
                 case "2":
-                    Console.WriteLine(" Not Implemented Yet \n");
+                    //File upload
+                    Console.WriteLine(" You choose 2, Upload File, in devl \n");
+                    FileUpload RemoteFileUPload = new FileUpload(conn);
                     MyAnswer = false;
                     break;
                 case "1":
-                    Console.WriteLine(" Not Implemented Yet  \n");
+                    //File download
```

Comment:
This works great but you should let the user know whether the download was successful or not. Similar to the above cases (delete from remote, create directory, etc.)